manifests/tasks/kserve-download-model.yaml

```
@@ -12,6 +12,8 @@ spec:
     type: string
   - name: s3-bucket-name
     type: string
+  - name: model-relative-path
+    tpye: string
```

This is why it failed:
```suggestion
    type: string
```